Shared/Samples/Show coordinates in multiple formats/ShowCoordinatesInMultipleFormatsView.swift

```diff
@@ -0,0 +1,179 @@
+                    fromLatitudeLongitudeString: model.latLongDMSString,
+                    spatialReference: model.mapPoint.spatialReference
+                ) {
+                    model.updateMapPoint(point: point)
+                } else {
+                    model.updateMapPoint(point: model.mapPoint)
+                }
+            }
+            CoordinateTextField(
+                title: "UTM",
+                text: $model.utmString
+            )
+            .onSubmit {
+                if let point = CoordinateFormatter.point(
+                    fromUTMString: model.utmString,
+                    spatialReference: model.mapPoint.spatialReference,
+                    conversionMode: .latitudeBandIndicators
+                ) {
+                    model.updateMapPoint(point: point)
+                } else {
+                    model.updateMapPoint(point: model.mapPoint)
+                }
+            }
+            CoordinateTextField(
+                title: "USNG",
+                text: $model.usngString
+            )
+            .onSubmit {
+                if let point = CoordinateFormatter.point(
+                    fromUSNGString: model.usngString,
+                    spatialReference: model.mapPoint.spatialReference
+                ) {
+                    model.updateMapPoint(point: point)
+                } else {
+                    model.updateMapPoint(point: model.mapPoint)
+                }
+            }
+            
+            MapView(map: model.map, graphicsOverlays: [model.graphicsOverlay])
+                .onSingleTapGesture { _, mapPoint in
+                    model.updateMapPoint(point: mapPoint)
+                }
+        }
+    }
+}
+
+struct CoordinateTextField: View {
+    /// The title of the text field.
+    var title: String
+    
+    /// The text in the text field.
+    @Binding var text: String
+    
+    var body: some View {
+        VStack(alignment: .leading) {
+            Text(title)
+                .padding([.leading, .top], 8)
+                .padding(.bottom, -5)
+            TextField("", text: $text)
+                .textInputAutocapitalization(.never)
+                .disableAutocorrection(true)
+                .textFieldStyle(.roundedBorder)
+                .padding([.leading, .trailing, .bottom], 8)
+        }
+    }
+}
+
+private extension ShowCoordinatesInMultipleFormatsView {
+    // The view model for the sample.
+    class Model: ObservableObject {
+        /// A map with an imagery basemap.
+        var map = Map(basemapStyle: .arcGISImageryStandard)
+        
+        /// The graphics overlay for the point graphic.
+        let graphicsOverlay: GraphicsOverlay
+        
+        /// A yellow cross graphic for the map point.
+        private let pointGraphic = Graphic(symbol: SimpleMarkerSymbol(style: .cross, color: .yellow, size: 20))
+        
+        /// The point on the map.
+        @Published var mapPoint: Point!
```

Comment:
A second thought: I feel it is better to be a state variable in the view than a published property in the model.

1. It doesn't drive UI updates, so it doesn't need to be published
2. I feel in most cases the published properties are private(set), public get. If it is both public set and get, then just staying in the view is fine.
3. If this suggestion is applied, method signature can change to `private func updateCoordinateFields(mapPoint: Point)`.